tests/test_ratmama.py

```diff
@@ -0,0 +1,131 @@
+"""
+
+TEST will be added laters
+"""
+
+import pytest
+from Modules.rat_board import RatBoard
+from tests.mock_callables import AsyncCallableMock
+from Modules.context import Context
+import Modules.RatMama as RatMama
+from tests.mock_callables import InstanceOf
+from Modules.rat_rescue import Platforms
+
+pytestmark = pytest.mark.ratmama
+
+
+@pytest.mark.asyncio
+async def test_ratmama_announcement(rat_board_fx: RatBoard, async_callable_fx: AsyncCallableMock,
+                                    context_channel_fx: Context, monkeypatch):
+    async def f(msg: str):
+        print(f"would have replied with: {msg}")
+        await async_callable_fx(msg)
+
+    monkeypatch.setattr(context_channel_fx, 'reply', f)
+    RatMama.board = rat_board_fx
+
+    monkeypatch.setattr(context_channel_fx, '_words_eol',
```

Comment:
if you are going to oveerwrite the content of the fixture, may as well not use the fixture.
I instead suggest using `Context.from_message`